src/Helper/ProcessHelper.php

```diff
@@ -80,7 +80,10 @@ public function runCommand($command, $allowFailures = false, $callback = null, $
         $process = $builder->getProcess();
 
         $remover = function ($untrimmed) {
-            return ltrim(rtrim($untrimmed, "'"), "'");
+            $removedSingleQuotes = ltrim(rtrim($untrimmed, "'"), "'");
+            $removedDoubleuotes = ltrim(rtrim($removedSingleQuotes, '"'), '"');
```

Comment:
@sstok should we then not do this change? since you say that it is how it is passed to the command?